library/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "library",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@slack/events-api": "^2.3.4",
+    "@slack/web-api": "^5.12.0"
+  }
+}
```

Comment:
maybe I should also move the discord bot code here, so the lib code would be separated from the backend service.

Reply:
yeah didn't start on discord yet. we should follow professor's direction and just make a separate repo for the library didn't think we could make more repos here but we can i